src/backend/distributed/utils/colocation_utils.c

```diff
@@ -988,7 +988,6 @@ DeleteColocationGroup(uint32 colocationId)
 	{
 		simple_heap_delete(pgDistColocation, &(heapTuple->t_self));
 
-		CatalogUpdateIndexes(pgDistColocation, heapTuple);
```

Comment:
It appears that postgres source code does not use`CatalogUpdateIndexes ()` after `simple_heap_delete ()` as far as I can see.